guis/qml/qml/MainQML.qml

```diff
@@ -30,8 +30,40 @@ import QtMultimedia 5.8
       // interceptor to make the registration captcha work
       var msg = "[Axolotl Web View] [JS] url changed %1".arg(url)
       console.log(msg)
-      var  interceptor = "window.onToken = function(token){window.location = 'http://localhost:9080/?token='+token;};"
-      webView.runJavaScript(interceptor)
+      if (url == "https://signalcaptchas.org/registration/generate.html"){
+        var  interceptor = `
+        // override the default onload function
+				
+				window.onload=function() {
+					var action = 'registration';
+					var isDone = false;
+					var sitekey = '6LfBXs0bAAAAAAjkDyyI1Lk5gBAUWfhI_bIyox5W';
```

Comment:
I am not sure of the importance of this key, but I see it three time: does it make sense/is is possible to extract it and only define it once for qml/qml, qml/ut and main.go? Not sure, but perhaps we can set it in the `~/.config/textsecure.nanuc/settings.yml` perhaps.

It would make it much easier to update it in the future for example.